tests/EdFi.TestFixture/DbContextExtensions.cs

```diff
@@ -23,16 +22,11 @@ public static void CatchAndWriteDbValidationExceptions(Action action)
             {
                 action();
             }
-            catch (DbEntityValidationException exception)
+            catch (ValidationException exception)
             {
-                var errorTexts =
-                    exception.EntityValidationErrors.SelectMany(x => x.ValidationErrors)
-                        .Select(x => x.ErrorMessage);
+                var errorTexts = exception.ValidationResult.ErrorMessage;
```

Comment:
Is this no longer an array? If that is the case, it may make sense to rename the variable to `errorText` (singular), or just get rid of the variable altogether and simply use `Console.WriteLine(exception.ValidationResult.ErrorMessage);`.